### PR TITLE
Dltp 1758 oai

### PR DESCRIPTION
**Pull request by laritakr**
Completes a portion of https://jira.library.nd.edu/browse/DLTP-1758
 
ListRecords verb is still unsupported, waiting for some changes in the oai gem. 
Additional fields can easily be added to the response by updating the field list in `app/models/oai_provider/curate_format.rb`

### CODE REVIEW

File: app/models/oai_provider/curate_format.rb [hunk 1]
```diff
@@ -0,0 +1,27 @@
+class OaiProvider < OAI::Provider::Base
```
Comment (jeremyf) [inline app/models/oai_provider/curate_format.rb]: How would you feel about moving the CurateFormat class and CurationConcernProvider class into the `app/models/oai_provider.rb` I don't see a strong reason to keep these files separate.

Comment (laritakr) [reply app/models/oai_provider/curate_format.rb]: I would be fine with that. I started out thinking more files would be needed but it ended up being pretty minimal.

Comment (jeremyf) [reply app/models/oai_provider/curate_format.rb]: Feel free to consolidate, I'll continue my review of what you have.

**Comment by jeremyf** [comment]
Looks good, I'll "sign off" after the consolidation.

File: app/models/oai_provider/curate_format.rb (hunk 1 above)
Comment (laritakr) [reply app/models/oai_provider/curate_format.rb]: Done

**Pull request merged**